### DISCUSSION TITLE
networking/ingress-nginx/high-concurrency

### DISCUSSION THREAD

**Discussion by giscus** [imroc/tke-guide#8] [General]
# networking/ingress-nginx/high-concurrency

概述

https://imroc.cc/tke/networking/ingress-nginx/high-concurrency

**Comment by ermazi** [comment]
"在高并发场景可能会导致 CPU 占用较高" 细说.
---
既然 stdout 经过容器转一层有较高的 CPU 损耗, 那为啥线上还是要建议打标准输出呢? 这能算最佳实践吗?

**Reply by imroc** [reply]
一般业务日志打印速度没那么快, 用stdout没问题,只有特别夸张的日志输出考虑不用stdout, 比如几十兆每秒的输出

**Reply by ermazi** [reply]
实际生产情况下, 没法控制啊, 稍微大点的节点上, 总日志量每秒上百兆很常见啊
